wire-ios-sync-engine/Source/SessionManager/APIVersionResolver.swift

```diff
@@ -99,8 +99,7 @@ final class APIVersionResolver {
         } else if
             isDeveloperModeEnabled,
             let preferredAPIVersion = BackendInfo.preferredAPIVersion,
-            allBackendVersions.contains(preferredAPIVersion)
-        {
+            allBackendVersions.contains(preferredAPIVersion) {
             WireLogger.environment.info("resolving to preferred api version \(preferredAPIVersion.rawValue)")
             BackendInfo.apiVersion = preferredAPIVersion
         } else if let apiVersion = commonProductionVersions.max() {
```

Comment:
For me the indentation of if and the block are difficult to read for the eye... maybe a blank line can help?